FFmpeg.AutoGen.CppSharpUnsafeGenerator/Generator.cs

```diff
@@ -208,16 +209,26 @@ private ASTContext ParseInternal(string[] sourceFiles)
                 LanguageVersion = LanguageVersion.C99_GNU
             };
 
-            parserOptions.SetupMSVC(VisualStudioVersion.Latest);
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
We can do generation now on windows only, if you like to prevent others from trying may be you can invert the check and throw not supported - platform  exception instead of commented code and todos?

Reply:
It throws PlatformNotSupportedException in else branch,  I comment out linux branch after realizing it's not supported. Do you want me to remove it instead?